tools/congestion-model/src/main.rs

```diff
@@ -0,0 +1,72 @@
+use congestion_model::designs::{GlobalTxStopShard, NoQueueShard, SimpleBackpressure};
+use congestion_model::workload::{AllForOneProducer, BalancedProducer, Producer};
+use congestion_model::{summary_table, Model, Shard, PGAS};
+
+fn main() {
+    let num_shards = 4;
```

Comment:
Is the plan to make this a CLI flag? Because otherwise I would probably prefer it a constant in `lib.rs` alongside the `GAS_LIMIT` et al.

Reply:
Most likely it will be a CLI flag, yes. Or maybe we will loop through a couple of numbers that we want to evaluate each time. But in case, we will test for several number of shards rather than just for a constant.